modules/op-about-triggers.adoc

```diff
@@ -106,18 +108,39 @@ metadata:
   name: vote-trigger <3>
 spec:
   serviceAccountName: pipeline <4>
+  interceptors:
+    - ref:
+        name: "github" <5>
+      params: <6>
+        - name: "secretRef" 
+          value:
+            secretName: github-secret
+            secretKey: secretToken
+        - name: "eventTypes" 
+          value: ["push"]
   bindings:
-    - ref: vote-app <5>
-  template: <6>
+    - ref: vote-app <7>
+  template: <8>
      ref: vote-app
+...
+apiVersion: v1
+kind: Secret <9>
+metadata:
+  name: github-secret
+type: Opaque
+stringData:
+  secretToken: "1234567"
 ----
 +
 <1> The API version of the `Trigger` resource. In this example, `v1alpha1`.
 <2> Specifies the type of Kubernetes object. In this example, `Trigger`.
 <3> Unique name to identify the `Trigger` resource.
 <4> Service account name to be used.
-<5> Name of the `TriggerBinding` resource to be connected to the `TriggerTemplate` resource.
-<6> Name of the `TriggerTemplate` resource to be connected to the `TriggerBinding` resource.
+<5> Interceptor name to be referenced. In this example, `github`.
```

Comment:
Just saying - please check the line spacing between <5>, <6>, and <7>.

Reply:
It is appearing correctly for me and I am unable to detect any issues. 

Can you please clarify what is the issue that you are seeing with line spacing?

Reply:
Line spacing appears all right both in the source and preview. Resolving the conversation